### PR TITLE
Grab artifacts from permission-slip

### PR DESCRIPTION
**Pull request by labbott**

### CODE REVIEW

File: .github/buildomat/jobs/tuf-repo.sh [hunk 1]
```diff
@@ -105,94 +100,28 @@ path = "/input/host/work/helios/upload/os-$kind.tar.gz"
...
 
-# Fetch SP images from a Hubris release.
+download_region_manifests() {
+	url=$1
+	name=$2
+	mkdir $2
+	pushd $2
+	while read -r line; do
+		manifest_hash=`echo $line | cut -d ' ' -f 1`
+		manifest_name=`echo $line | cut -d ' ' -f 2`
```
Comment (iliana) [inline .github/buildomat/jobs/tuf-repo.sh:110]:
```suggestion
	while read -r manifest_hash manifest_name; do
```

```
$ while read -r manifest_hash manifest_name; do echo "[$manifest_hash][$manifest_name]"; done <tools/permslip_staging
[bbc8ff243b77d7f1d3468ad063f1c42af0e059345dcee6388f15c398029efa03][manifest-gimlet-v1.0.14.toml]
[2eede10ac4e236ba777918dd456427d0e161293492f5e1c4165c199e5bec3fc6][manifest-oxide-rot-1-v1.0.8.toml]
[e5aaba5e504e887f7897e753881ae84ee2db8784e60410539d0ee3e7f93a26b1][manifest-psc-v1.0.13.toml]
[4128d9a93f3cd1ff2d8317815bcf12050ad0f3ea8bdf2e86e3736be732a0cb39][manifest-sidecar-v1.0.13.toml]
```

**Review by iliana — APPROVED** [review]
Shell script changes seem overall fine. I presume the remaining changes are getting prod builds signed and listing their manifests in `tools/permslip_production`.

**Review by iliana — APPROVED** [review]
Checked against a tufaceous manifest from main to make sure we didn't drop any artifacts, looks good.

**Comment by labbott** [comment]
Marking as draft again because the Gimlet image in this repo fails to boot and we need to do another one (but the TUF repo itself seems to work fine)

**Pull request merged**